src/data/matrix/basic.lean

```diff
@@ -1442,6 +1442,12 @@ lemma conj_transpose_reindex [has_star α] (eₘ : m ≃ l) (eₙ : n ≃ o) (M
   (reindex eₘ eₙ M)ᴴ = (reindex eₙ eₘ Mᴴ) :=
 rfl
 
+@[simp]
+lemma minor_mul_transpose_minor [fintype n] [fintype m] [semiring α]
+  (e : n ≃ m) (M : matrix n m α) :
+  (M.minor id e) ⬝ (Mᵀ).minor e id = M ⬝ Mᵀ :=
+by rw [←minor_mul_equiv, minor_id_id]
```

Comment:
Is this evidence that `minor_mul_equiv` should be stated in reverse and marked simp?

Reply:
Let me see if it breaks a lot of things.

Reply:
I don't know if stating `minor_mul_equiv` in reverse is better or not, but I think that in any case it is something for another PR, since one should modify also `reindex_linear_equiv_mul` and so on.

Reply:
Sorry, I didn't mean that as a suggestion for _this_ PR!

Reply:
I think reversing `minor_mul_equiv` is probably a good idea. I've opened #10638 so we don't forget.